### PR TITLE
responding to issue #8311

### PR DESCRIPTION
**Pull request by sanbrock**
Make updater more annoying #8311
Rename "skip this version" to "Don't ask again for one week"
=======================================

+ Buttons are renamed, but translation files shall also be modified/extended.
+ Another config file setting seenVersionDate has been introduced in which the last date this version was seen /asked to be skipped/ is stored/updated. The new entry in the config file looks like:
[Updater]
seenVersion=3.1.0.3100
seenVersionDate=2021-03-05T18:07:14Z

+ This setting (if exists) is checked against actual date before accepting to (temporarily) skip this update.  Note that the validity of the stored date is also checked (if it was not set to a future date to avoid being annoying)

### CODE REVIEW

File: src/gui/updater/ocupdater.cpp [hunk 1]
```diff
@@ -469,8 +481,10 @@ void NSISUpdater::showUpdateErrorDialog(const QString &targetVersion)
     connect(retry, &QAbstractButton::clicked, msgBox, &QDialog::accept);
     connect(getupdate, &QAbstractButton::clicked, msgBox, &QDialog::accept);
 
+    //"skipping" is now asking again after 1 week
     connect(skip, &QAbstractButton::clicked, this, [this]() {
-        wipeUpdateData();
+        //wiping is temporarily removed until version skipping is properly fixed
```
[the remark this answers is not in the document]
Comment (sanbrock) [reply src/gui/updater/ocupdater.cpp]: it was not clear, but removed after clarifying that removing the possibility of skipping versions is a longer term intention.

**Comment by MarDiehl** [comment]
I understand the intention of nudging the user to update the client, but as an Arch linux user with rolling release it is superfluous and annoying. The package maintainer takes care of the update. Is there a way to completely disable the notifications?

**Comment by fmoc** [comment]
@MarDiehl Your package's maintainer can write a patch to deactivate it.

**Comment by MarDiehl** [comment]
> @MarDiehl your package's maintainer can write a patch to deactivate it.

thanks, I'll do that (or try to write a patch myself)

**Pull request closed without merging**